### ISSUE TITLE
Batch Explorer showing that the task is working but its not

### ISSUE THREAD

**Issue by Gaploid** [Azure/BatchExplorer#2024]
**Describe the bug**
The status for task sometimes shows that it working but it's not. Here is the example 
![image](https://user-images.githubusercontent.com/6171245/56387319-b4a05400-622c-11e9-9f90-f4d234e392b5.png)
The task already cleaned from the node but it still shows status - running.

**Comment by timotheeguerin** [comment]
hhm intresting, you mean the task should be in completed state or is deleted and should not show?

Does nothing happens when manually clicking refresh?
Is there also any errors in the logs? `Profile > View logs`

**Comment by Gaploid** [comment]
Hmm, it seems that's an Azure Batch bug cause on Azure portal the situation is the same.
![image](https://user-images.githubusercontent.com/6171245/56389898-63479300-6233-11e9-8640-cd4861c2e2b4.png)
Only 10 of these task in real "running" state. Others were completed and cleaned from the nodes.

**Comment by timotheeguerin** [comment]
I see, if its an issue with the service I would recommend opening a support request.

**Comment by timotheeguerin** [comment]
It wouldn't be a timing issue? when you refresh the page does the same task(That should be completed) still show as running or they are completed?

**Comment by Gaploid** [comment]
Nope, as you can see they sit in these states indefinitely (3 -5 - 9999999999 hours). 

I cant create support ticket because 
![image](https://user-images.githubusercontent.com/6171245/56390349-8161c300-6234-11e9-9866-56cb1a6f9126.png)
I don't think that I need to pay $29 to send a bug report and be a tester on behalf of the product team.

**Comment by timotheeguerin** [comment]
How long do you expect your task to run? We believe this is related to the nodes getting preempted

**Comment by Gaploid** [comment]
From 1 hour till 5 hours. If Vm was preempted then the task should go back to the queue automatically, or am I wrong?

**Comment by timotheeguerin** [comment]
This is an issue where the state will apparently stay in running when the node gets preempted but it will still go back in the queue and run as expected on another node later. @alfpark  can confirm.

**Comment by alfpark** [comment]
Correct, the task state will remain in running state if the VM is preempted. We realize this is not an optimal user experience, and we are working on improvements in this area.
